app/src/main/java/com/example/studentarena/fragments/SearchFilterFragment.java

```diff
@@ -30,6 +31,8 @@ public class SearchFilterFragment extends DialogFragment {
     private TextInputEditText textInputEditTextMin;
     private TextInputEditText textInputEditTextMax;
     private Button ibDone;
+    public static double maxPrice = 100000000;
+    public static double minPrice = 0;
```

Comment:
i think these should be `Double` instead of `double`. `Double` is one of those "boxed types" which provide methods like `toString()`, and also they allow you to use `null` to indicate that `minPrice` or `maxPrice` wasn't specified. if you change these to `Double` and initialize them as `null`, then you can wrap them in `if (minPrice != null) {}` checks like you do with the title